shanoir-ng-datasets/src/main/java/org/shanoir/ng/importer/service/BidsImporterService.java

```diff
@@ -266,8 +266,9 @@ private void importDataset(ImportJob importJob, BidsDataType bidsDataType, Datas
 		eventService.publishEvent(new ShanoirEvent(ShanoirEventType.CREATE_DATASET_ACQUISITION_EVENT, datasetAcquisition.getId().toString(), KeycloakUtil.getTokenUserId(null), "", ShanoirEvent.SUCCESS, examination.getStudyId()));
```

Comment:
Hi Youenn, very minor detail, but could you please change to getTokenUserId() and remove the "null" param, that looks quite strange. With kind regards, Michael